### ISSUE TITLE
Define CWR validation requirements for Rights Societies

### ISSUE THREAD

**Issue by revelator-labs** [originalworks/protocol#64]
- What version are we officially receiving? (currently validate to 2.1) 
- Who is the Rights Society sending the message - this will define the sender as the Rights Controller
- The right type that will be monetized via Original Works will be fixed to "Public Performance Rights"
- Share percentage is collected by the rights controller (sender)
- Territory in which the royalties for the specified rights types are being collected
- If ISRC is also specified it is indexed as metadata but not registered as the official verified ISRC

**Comment by revelator-labs** [comment]
JSON version of the CWR received from Socinpro
[cwr_json.txt](https://github.com/user-attachments/files/17306093/cwr_json.txt)

**Comment by revelator-labs** [comment]
OWR should be the Original Writer - The parser needs to be fixed, as it mistakingly recorded the original writer

If the "Original Writer" (ORW) is specified but the "shares" number does not start with the 4 digit group_id then do not register those shares, but only capture the "Original Writer" `name` as part of the asset meta data.

**Comment by criadoperez** [comment]
I think its the last 5 digits, not 4.  So `18901250` means `012.50%`. And `10000` (100%) is the maximum value. 
`189` is the` group_id` code in the example

**Comment by revelator-labs** [comment]
Correct. I've fixed the comment to reflect the group_id regardless of its number of digits.

**Comment by criadoperez** [comment]
Received first CWR on IPFS.
Pending conversion to JSON format

**Comment by criadoperez** [comment]
Pending to receive more CWR file to start processing them.

**Comment by criadoperez** [comment]
we will apply `mechanical rights` worldwide

**Comment by criadoperez** [comment]
Pending meeting with first CWR data provider to confirm data format

**Comment by criadoperez** [comment]
Meeting second week of March. Update expected right after.
